client/components/fraud-risk-tools-banner/index.tsx

```diff
@@ -27,29 +24,15 @@ const FRTDiscoverabilityBanner: React.FC = () => {
 		try {
 			return JSON.parse( frtDiscoverBannerSettings );
 		} catch ( e ) {
-			return { remindMeCount: 0, remindMeAt: null, dontShowAgain: false };
+			return { dontShowAgain: false };
```

Comment:
Now that we are holding just a single piece of information, what do you think about changing `frtDiscoverBannerSettings` to `frtDiscoverBannerDismissed` that will hold a boolean?

In that way, we won't have to have all the `JSON.parse` and `JSON.stringify` logic.

Reply:
This is a good idea. I've made the changes in [ab8055f](https://github.com/Automattic/woocommerce-payments/pull/6664/commits/ab8055f5a75869456562f006d840b4a310fe5402). Unit tests and my manual tests pass. Could you take a look and see if the code looks how you expect?

Reply:
I made some minor changes in [c26b987](https://github.com/Automattic/woocommerce-payments/pull/6664/commits/c26b987677cc92ae7c8e9376741f9f8296abee0c).